static/app/views/performance/browser/resources/shared/constants.ts

```diff
@@ -1 +1,2 @@
 export const FONT_FILE_EXTENSIONS = ['woff', 'woff2', 'ttf', 'otf', 'eot'];
+export const IMAGE_FILE_EXTENSIONS = ['jpg', 'jpeg', 'png', 'gif', 'svg', 'webp'];
```

Comment:
Let's add [`apng`](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#apng_animated_portable_network_graphics) and [`avif`](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#avif_image)

Reply:
👍